bin/push-templates-json.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+process.removeAllListeners('warning');
+
+// ES Modules
+import {createRequire} from 'module'
+import {fileURLToPath} from "node:url"
+import * as path from "pathe"
+
+// CJS Modules
+const require = createRequire(import.meta.url)
+const {readFile} = require('fs-extra')
+const {Octokit} = require('@octokit/core')
+const {createPullRequest} = require("octokit-plugin-create-pull-request")
+const colors = require('ansi-colors')
+
+const fileToPush = path.resolve(path.dirname(fileURLToPath(import.meta.url)), "../templates.json")
+console.log("Opening a PR to update the templates.json file on CDN")
+
+const files = {
+  "static-24h/cli/extensions/templates.json": await readFile(fileToPush)
+}
+
+const OctokitWithPlugin = Octokit.plugin(createPullRequest)
+const octokit = new OctokitWithPlugin({
+  auth: process.env.GITHUB_TOKEN,
+});
+
+const response = await octokit
+  .createPullRequest({
+    owner: "Shopify",
+    repo: "static-cdn-assets",
+    title: `Shopify CLI - update extension templates - ${new Date().toISOString().split('T')[0]}`,
+    body: `We are updating the extension templates available for generating in apps via Shopify CLI`,
```

Comment:
Should we include some context here that this is an automated PR? Will it be obvious?

Reply:
This is exactly what we do for Homebrew, but I can add in something a bit more explicit.